crates/lowering/tests/snapshots/lowering__and_or.snap

```diff
@@ -0,0 +1,182 @@
+---
+source: crates/lowering/tests/lowering.rs
```

Comment:
general note on these snapshots: I think we should just include the unlowered code in the snapshot